config/webpack/webpack.config.static.js

```diff
@@ -69,7 +69,8 @@ module.exports = {
         head: {
           appendChild: function () {}
         }
-      }
+      },
+      Element: function () {}
```

Comment:
Ah, introduced by clipboard dep added to ecology: https://github.com/FormidableLabs/ecology/pull/45/commits/a741917e1e1e0b186558df72d8231ed68224ab4f